lamindb/curators/core.py

```diff
@@ -803,6 +870,7 @@ def __init__(
         feature: Feature | None = None,
         cat_manager: DataFrameCatManager | None = None,
         subtype_str: str = "",
+        maximal_set: bool = False,
```

Comment:
A comment would be good here because it's subtly different from what it means on `Schema`. But because the arg is so generically named it's still a good name also here.